app/pkcs.py

```diff
@@ -129,6 +125,34 @@ def getf9(self, slot):
         return b64crt
 
     def makecsr(self, session, private_key, public_key):
+        from asn1crypto import csr as csr_module, x509
+
+        san = x509.GeneralNames(
+            [
+                x509.GeneralName("dns_name", "example.com"),
+                x509.GeneralName("dns_name", "www.example.com"),
+            ]
+        )
+        extensions = [
+            csr_module.CRIAttribute(
+                {
+                    "type": "extension_request",
+                    "values": [
+                        x509.Extensions(
+                            [
+                                x509.Extension(
+                                    {
+                                        "extn_id": "subject_alt_name",
+                                        "critical": False,
+                                        "extn_value": san,
+                                    }
+                                )
+                            ]
+                        )
+                    ],
+                }
+            )
+        ]
```

Comment:
This should normally not be needed for our CSR, but is needed for the implementation in ACME CA Server.